core/ir/x86/opcode_api.h

```diff
@@ -1614,6 +1615,9 @@ enum {
     /* 1433 */ OP_vpopcntd, /**< IA-32/AMD64 vpopcntd opcode. */
```

Comment:
You've overlooked step 8 on the list at the top of this file.  Please enable the currently disabled lines in the binutils tests:

```
$ git grep 'RAW(0f) RAW(01) RAW(c[ab]'
third_party/binutils/test_decenc/test_decenc_x86.asm:        /* RAW(0f) RAW(01) RAW(ca) */
third_party/binutils/test_decenc/test_decenc_x86.asm:        /* RAW(0f) RAW(01) RAW(cb) */
third_party/binutils/test_decenc/test_decenc_x86_64.asm:        /* RAW(0f) RAW(01) RAW(ca) */
third_party/binutils/test_decenc/test_decenc_x86_64.asm:        /* RAW(0f) RAW(01) RAW(cb) */
```

Reply:
Yeah I completely missed those steps.

Reply:
Because there are some pc-relative instrs also, simply uncommenting those lines led to a huge diff. For now I moved them to the end of the file and added a TODO to move them back in a separate PR to keep the huge diff isolated.